rhombus/rhombus/scribblings/reference/port.scrbl

```diff
@@ -24,8 +24,22 @@ output; it is possible for an object to be both an input and output port.
 
 @doc(
  def Port.eof :: Port.EOF
+ bind.macro 'Port.eof'
 ){
 
- A value (distinct from all other values) that represents an end-of-file.
+ The @rhombus(Port.eof) value represents an end-of-file (distinct from
+ all other values), and the @rhombus(Prof.eof) binding match matches that
```

Comment:
`Prof.eof` -> `Port.eof`

Reply:
Hah! It's retention-and-tenure-review season, so maybe my fingers have become mistrained.